tests/unit/api/bucketGetWebsite.js

```diff
@@ -0,0 +1,97 @@
+import assert from 'assert';
+
+import bucketPut from '../../../lib/api/bucketPut';
+import bucketPutWebsite from '../../../lib/api/bucketPutWebsite';
+import bucketGetWebsite from '../../../lib/api/bucketGetWebsite';
+import { cleanup,
+    DummyRequestLogger,
+    makeAuthInfo }
+from '../helpers';
+
+const log = new DummyRequestLogger();
+const authInfo = makeAuthInfo('accessKey1');
+const bucketName = 'bucketname';
+const locationConstraint = 'us-west-1';
+const testBucketPutRequest = {
+    bucketName,
+    headers: { host: `${bucketName}.s3.amazonaws.com` },
+    url: '/',
+};
+
+function _makeWebsiteRequest(xml) {
+    const request = {
+        bucketName,
+        headers: {
+            host: `${bucketName}.s3.amazonaws.com`,
+        },
+        url: '/?website',
+        query: { website: '' },
+    };
+
+    if (xml) {
+        request.post = xml;
+    }
+    return request;
+}
+const testGetWebsiteRequest = _makeWebsiteRequest();
+
+function _comparePutGetXml(sampleXml, done) {
+    const fullXml = '<?xml version="1.0" encoding="UTF-8" ' +
+    'standalone="yes"?><WebsiteConfiguration ' +
+    'xmlns="http://s3.amazonaws.com/doc/2006-03-01/">' +
+    `${sampleXml}</WebsiteConfiguration>`;
+    const testPutWebsiteRequest = _makeWebsiteRequest(fullXml);
+    bucketPutWebsite(authInfo, testPutWebsiteRequest, log, err => {
+        if (err) {
+            process.stdout.write(`Err putting website config ${err}`);
+            return done(err);
+        }
+        return bucketGetWebsite(authInfo, testGetWebsiteRequest, log,
+        (err, res) => {
+            assert.strictEqual(err, null, `Unexpected err ${err}`);
+            assert.strictEqual(res, fullXml);
+            done();
+        });
+    });
+}
+
+describe('getBucketWebsite API', () => {
+    before(() => cleanup());
```

Comment:
I must have based this off of another unit test, but technically I believe it should just be run in the afterEach() to clean up properly. It was probably a safeguard to take care of any other set of tests that hadn't cleaned up properly. (If I remove it without changing the bucket name used in the tests, I get a BucketAlreadyOwnedByYou error, which means some previously run test did not clear the list of buckets stored in memory). I'll remove it as it is unnecessary.